controllers/solrcloud_controller.go

```diff
@@ -63,18 +63,18 @@ func SetIngressBaseUrl(ingressBaseUrl string) {
 	IngressBaseUrl = ingressBaseUrl
 }
 
-// +kubebuilder:rbac:groups=,resources=pods,verbs=get;list;watch
-// +kubebuilder:rbac:groups=,resources=pods/status,verbs=get
-// +kubebuilder:rbac:groups=,resources=services,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=,resources=services/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups="",resources=pods,verbs=get;list;watch
```

Comment:
this can't be blank any longer, we must explicitly express "" for system level api access for things like pods

Reply:
interesting. Good to know